wiki/Player-Conditions.md

```diff
@@ -177,9 +177,11 @@ No error will be raised if you modify these conditions, but the game will reset
 
 ## Value expressions
 
-Beginning with **v0.9.11**, support was added for simple algebra in both types of conditions. For [testable](#testable-condition-sets) conditions, these "value expressions" can appear on either side of the comparison operator, while [applied](#applied-condition-sets) conditions can only use value expressions on the right-hand side of the mutation operator. (This is because an applied condition must store the condition value with a name, and the result of evaluating a value expression is an integer, not a name.)
+Beginning with **v0.9.11**, support was added for simple algebra in both types of conditions (and this was extended with boolean operators and built-in functions in **v0.10.15**). For [testable](#testable-condition-sets) conditions, these "value expressions" can appear on either side of the comparison operator, while [applied](#applied-condition-sets) conditions can only use value expressions on the right-hand side of the mutation operator. (This is because an applied condition must store the condition value with a name, and the result of evaluating a value expression is an integer, not a name.)
```

Comment:
```suggestion
Beginning with **v0.9.11**, support was added for simple algebra in both types of conditions (and this was extended with boolean operators and built-in functions in **v0.10.17**). For [testable](#testable-condition-sets) conditions, these "value expressions" can appear on either side of the comparison operator, while [applied](#applied-condition-sets) conditions can only use value expressions on the right-hand side of the mutation operator. (This is because an applied condition must store the condition value with a name, and the result of evaluating a value expression is an integer, not a name.)
```